jest.config.js

```diff
@@ -1 +1,4 @@
-module.exports = {transform: {'.js$': '<rootDir>/scripts/babel-jest-transformer.js'}}
+module.exports = {
+  testEnvironment: 'node',
+  transform: {'.js$': '<rootDir>/scripts/babel-jest-transformer.js'},
```

Comment:
Use the test environment config in `.babelrc` instead of custom script

https://github.com/facebook/jest#using-babel